Empty.bot

```diff
@@ -0,0 +1,8 @@
+state goNorthEast
+  Nopen -> go North
+  Nclosed, Eopen -> go East
+  Any -> turn West, transition Sweep
```

Comment:
I wonder if Nopen and Nclosed are really the best words here. They're definitely unambiguous, but also sort of long.